compiler-cli/src/new.rs

```diff
@@ -12,10 +12,10 @@ use strum::{Display, EnumString, EnumVariantNames};
 use crate::NewOptions;
 
 const GLEAM_STDLIB_VERSION: &str = "0.25";
-const GLEEUNIT_VERSION: &str = "0.7";
-const ERLANG_OTP_VERSION: &str = "25.1";
-const REBAR3_VERSION: &str = "25.1";
-const ELIXIR_VERSION: &str = "1.14.1";
+const GLEEUNIT_VERSION: &str = "0.7.2";
```

Comment:
Sorry, could you undo this also please? We do not pin hex libraries to patch updates only

Reply:
ok

Reply:
done.